electrumsv/wallet.py

```diff
@@ -164,7 +160,7 @@ def set_key_history(self, key_id: int,
 
         for tx_id in added_tx_ids:
             if tx_id not in self._tx_keys:
-                 self._tx_keys[tx_id] = set()
+                self._tx_keys[tx_id] = set()
```

Comment:
Didn't you replace `_tx_keys` with `_script_keys`? How is this not erroring if so?

Reply:
That's just fixing indentation. Nothing to do with any of the other changes. Had me worried!